static/app/components/charts/chartZoom.tsx

```diff
@@ -256,7 +255,9 @@ class ChartZoom extends Component<Props> {
       });
     }
 
-    callIfFunction(this.props.onFinished);
+    if (typeof this.props.onFinished === 'function') {
+      this.props.onFinished(_props, chart);
+    }
```

Comment:
Should we still be doing the `typeof` check, or is `this.props.onFinished?.(_props, chart)` enough because of the prop types

Reply:
We had a chat with @evanpurkhiser a while back. I think use whatever you want. I personally find it easier to read a  if statement + typeof operator as I think it makes it more explicit that it's a conditional call, but if someone wants to use ?.() I wont mind. I used to always write ?.(), but then sometimes I noticed when reading the code, my brain would skip ?.() and I wouldn't notice it right away so I just opt to write it 🤷🏼